packages/devel/patchelf/package.mk

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: GPL-2.0-only
+# Copyright (C) 2023-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="patchelf"
+PKG_VERSION="0.18.0"
+PKG_SHA256="1952b2a782ba576279c211ee942e341748fdb44997f704dd53def46cd055470b"
+PKG_LICENSE="BSD"
```

Comment:
https://github.com/NixOS/patchelf#license GLP3 or later ?

Reply:
Not sure how I got that wrong. Fixed.